src/components/Images/Images.js

```diff
@@ -1,31 +1,17 @@
-import React from 'react';
-import { Link } from 'react-router';
+import React, { PropTypes as T } from 'react';
 
-const Images = ({ photos }) => (
+import Image from './Image';
+
+export default function Images({
```

Comment:
Great minds think alike. I changed that last night myself and was going to submit a pull request today! I agree, especially after talking to Purvi. I think we should use Class components or function components.